app/components/back-office-app-services/variables.tf

```diff
@@ -244,6 +244,12 @@ variable "servicebus_topic_nsip_documents_id" {
   type        = string
 }
 
+variable "servicebus_topic_register_nsip_subscription_id" {
+  description = "Service Bus Topic register-nsip-subscription id"
+  type        = string
+  default     = ""

```

Comment:
I think let's remove this default because the component should fail to initialise without it - it should be ok since you're explicitly passing in an empty string in uk-south. It's not any different to what you have, but probs easier to refactor later on.

Reply:
Greed, that makes sense.